data-tidy.qmd

```diff
@@ -1,31 +1,31 @@
-# Data tidying {#sec-data-tidy}
+# Tornando os dados *tidy* {#sec-data-tidy}
```

Comment:
**Provisório**. Os títulos dessas seções quase sempre fazem uma brincadeira com "*data \<something\>*", creio que seria interessante pensar em uma forma de consolidar todos os título semelhantes em português para manter a brincadeira.

Esse título aí é o mais "explicativo", mas eu to aberto a consolidarmos uma versão **oficial** de "tidy" quando utilizado como **verbo** e quando utilizado como **substantivo**. Isso provavelmente vai gerar uma pá de ajustes na tradução abaixo, mas acho que a gente mata pro resto do livro.

Outras sugestões para esse mesmo título:
- **Arrumando os dados** *(ruim pq não aparece "tidy" no título)*
- **Organizando os dados** *(idem)*
- *qualquer uma das anteriores + o título original em parênteses*

Reply:
- Organizando os dados (data tidying) ?

Reply:
Embora a sua sugestão inicial não me parece ruim